build.gradle

```diff
@@ -34,6 +36,8 @@ configurations {
 
 dependencies {
     compile gradleApi()
+    compileOnly 'org.projectlombok:lombok:1.18.8'
```

Comment:
I like lombok, but it's not worth it to add this to remove 4 getters.

Reply:
Yes, but I like to keep the code cleaner. Lombok provides the future possibility to remove a ton of boilerplate code present in the plugin.